doc/project/testing.md

```diff
@@ -74,3 +74,14 @@ for example `sbt> fooJS/it:test`.
 A list of testing frameworks compatible with Scala.js can be found [here](../../libraries/testing.html).
 
 Note: Don't forget to mark a test framework SBT dependency as `test,it` if you have both unit and integration tests.
+
+## Testing over `fullOptJS`-generated files
+
+By default, tests runs over `fastOptJS`-built JS files since their build time are shorter than `fullOptJS`.
+
+If you want to run tests over `fullOptJS`-build JS files, run `set scalaJSStage in Global := FullOptStage` before test.
+Or you can set it in a project settings in sbt file.
```

Comment:
I would not recommend doing that. When you do that, you don't test `fastOpt` anymore, which means that you'll miss all the checks for undefined behavior, and then you'll think that your code was correct when in fact it wasn't.

Not to mention the significantly longer times necessary to test.

Testing in fullOpt should be left to the CI 99.9% of the time, and so not be in your build as the default thing to do.

Reply:
Updated https://github.com/scala-js/scala-js-website/pull/456/commits/6c01edcd89493aea92f0272ebafd1bb348c78f2f